server/src/RestResourceHost.cc

```diff
@@ -487,6 +488,13 @@ HatoholError RestResourceHost::parseEventParameter(EventsQueryOption &option,
 		option.setTriggerStatuses(statuses);
 	}
 
+	// countOnly
+	value = static_cast<const gchar*>(
+	  g_hash_table_lookup(query, "countOnly"));
+	if (value && *value && string(value) == "true") {
```

Comment:
How about returning an error or logging that when the RHS of countOnly is neither "true" or false ?